doc/plugins/proxy-rewrite-cn.md

```diff
@@ -29,7 +29,6 @@
 |uri             |可选| 转发到上游的新 `uri` 地址|
 |regex_uri       |可选| 转发到上游的新 `uri` 地址, 使用正则表达式匹配来自客户端的uri，当匹配成功后使用模板替换转发到上游的uri, 未匹配成功时将客户端请求的uri转发至上游。当`uri`和`regex_uri`同时存在时，`uri`优先被使用。例如：["^/iresty/(.*)/(.*)/(.*)","/$1-$2-$3"] 第一个元素代表匹配来自客户端请求的uri正则表达式，第二个元素代表匹配成功后转发到上游的uri模板|
 |host            |可选| 转发到上游的新 `host` 地址，例如：`iresty.com` |
-|enable_websocket|可选| 是否启用 `websocket`（布尔值），默认不启用|
```

Comment:
you not remove the code in `proxy-rewrite.lua`

Reply:
we can do this in a new separate PR

Reply:
please keep the doc and code are the same.

Reply:
sure, will revert proxy-write's doc and change it in another PR